package.json

```diff
@@ -589,6 +589,11 @@
                     "type": "boolean",
                     "default": false,
```

Comment:
Also same deal from the other PR - udpate the reference to the appservice package version since this PR directly relies on it